constants.js

```diff
@@ -108,6 +108,9 @@ const constants = {
     legacyLocations: ['sproxyd', 'legacy'],
     /* eslint-disable camelcase */
     externalBackends: { aws_s3: true, azure: true },
+    // some of the available data backends need a key provided as a string
```

Comment:
"some of the available data backends (if called directly rather than through the multiple backend gateway) need..."